### DISCUSSION TITLE
Installing SSL Certs, using my own Certificate Authority getting a error

### DISCUSSION THREAD

**Discussion by JakeTrans** [AdguardTeam/AdGuardHome#6828] [Q&A] [answered]
Hi,

I'm getting a "certificate does not verify: x509: certificate signed by unknown authority" when trying to add my own certificate signed by my own authority.  I'm using Adguard without a domain so the LetEncrypt  solution wouldn't work for me and I'm running Adguard as part of a internal Homelab, how can I add my CA to be a known authority as this is the last server on on my homelab to be setup with a SSL Cert? 

or is it not possible to add a SSL certificate via your own authority?

I'm running Adguard in Docker on a X64 Intel computer using Debian if helps.

Thanks you for any help you can give.

**Comment by ghost** [comment, marked as answer]
Have you added the CA's root certificate to the trusted store on Debian?

You'll need to add the CA root certificate to the Docker image to `/usr/local/share/ca-certificates/`, set the permissions to `644`, then update the CA store inside the container with the command `update-ca-certificates`.

**Reply by JakeTrans** [reply]
Hi, 

Thanks for replying

The CA is in the trusted store on the Debian box - I'll try updating the Root Certs in the container

Thanks for the help - I'll Update once I've tried

**Reply by JakeTrans** [reply]
It works it was the certs where not in the folder for the CAs

for reference
to copy the certs
docker cp (CertFile).pem (ContainerName):/usr/local/share/ca-certificates/(CertFile).pem

to update the CAs
docker container exec -it (ContainerName) update-ca-certificates

Thanks for the help
